kidviz/views.py

```diff
@@ -492,14 +492,14 @@ def get_context_data(self, **kwargs):
             date_to = date_filtering_form.cleaned_data['date_to']
             selected_constructs = date_filtering_form.cleaned_data['constructs']
             tags = date_filtering_form.cleaned_data['tags']
-            course = date_filtering_form.cleaned_data['course']
+            courses = date_filtering_form.cleaned_data['courses']
 
             # If there aren't any query params use default course.
             if self.request.GET:
-                if course:
-                    course_id = course.id
+                if courses:
+                    course_id = [course.id for course in courses]
                 else:
-                    course_id = None
+                    course_id = (course_id or None)
```

Comment:
Why? `bool` function has the same result on empty list and None. Do you need this one to ui?

Reply:
Am yes but course_id may not be empty